lib/wrappers/transform-node.ts

```diff
@@ -18,7 +20,15 @@ export default class TransformNodeWrapper extends NodeWrapper {
       outputPath: this.outputPath,
       cachePath: this.cachePath,
     });
-    this.callbackObject = this.nodeInfo.getCallbackObject();
+    //adding ts-ignore for now. Should be removed later when new version of broccoli-node-api is released
+    // @ts-ignore
+    if (this.nodeInfo.fsFacade) {
+      this.callbackObject = this.nodeInfo.getCallbackObject();
+      this.callbackObject.input = new FSMerger(this.nodeInfo.inputNodes).fs,
```

Comment:
Why are we mutating the callbackObject that the user-land plugin gave to us?

Reply:
We do not have a method (not one that I know) to set these two properties into plugin other than mutating here.